examples/friends/test/friends_test.exs

```diff
@@ -0,0 +1,8 @@
+defmodule FriendsTest do
+  use ExUnit.Case
+  doctest Friends
+
+  test "the truth" do
+    assert 1 + 1 == 2
+  end
+end
```

Comment:
The idea of this `examples/friends` is to show the final point of the getting started guide. There are no tests in the guide, and this file isn't even mentioned so I haven't touched it.